hypha/settings/base.py

```diff
@@ -672,3 +684,19 @@
 PROJECTS_AUTO_CREATE = False
 if env.get('PROJECTS_AUTO_CREATE', 'false').lower().strip() == 'true':
     PROJECTS_AUTO_CREATE = True
+
+
+# django-file-form settings
+
+FILE_FORM_MUST_LOGIN = True
```

Comment:
>Must the user be logged in to upload a file.

I think this should be set as False as users without account can also apply to our forms.

Reply:
Sorry, I assumed it's logged in only.

Btw, I guess you might want to consider adding file size validation too. I've just tried a [large pdf](https://download.lenovo.com/pccbbs/mobiles_pdf/t480_hmm_en.pdf) and it returned 503 after about 20MB but there was no indication that it failed, the error appeared on the console only. (I'm on mobile data so this might have been a network error too.)

Reply:
>Sorry, I assumed it's logged in only.

No problem, I was just checking.

>Btw, I guess you might want to consider adding file size...

Thanks, will look into that.